docs/jwst/group_scale/description.rst

```diff
@@ -31,7 +31,7 @@ The step decides whether rescaling is necessary by comparing the
 values of the NFRAMES and FRMDIVSR keywords. If they are equal,
 then the on-board averaging was computed correctly and this step
 is skipped. In this case, the calibration step status keyword
-``S_GRPSCL`` is set to "SKIPPED". If the keyword values are not
+``S_GRPSCL`` is set to "SKIPPED." If the keyword values are not
```

Comment:
I know the rule in English is that a period or comma should be included within the quotes, but I think it's a mistake to follow that rule in this context, i.e. when specifying a value that one can expect to see in a file.

Reply:
fixed in many files